pgspecial/iocommands.py

```diff
@@ -35,7 +35,7 @@ def get_filename(sql):
 
 @export
 def get_watch_command(command):
-    match = re.match("(.*) \\\\watch (\d+);?$", command)
+    match = re.match("(.*?)[ \t\n]*\\\\watch (\d+);?$", command)
```

Comment:
Do you it's better to use `\s*` instead of `[ \t\n]*`? That way, in case someone uses other spaces (like `\r`, or non-ASCII spaces), it will still recognize it.